controller_manager:
  ros__parameters:
    # globalny update rate dla controller managera
    update_rate: 250

    # Kontroler ramienia
    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - shoulder_pan_joint
        - shoulder_lift_joint
        - elbow_joint
        - wrist_1_joint
        - wrist_2_joint
        - wrist_3_joint
      command_interfaces:
        - position
      state_interfaces:
        - position
        - velocity
      constraints:
        stopped_velocity_tolerance: 0.01
        goal_time: 3.0
      allow_partial_joints_goal: false

    # Kontroler chwytaka RG2
    gripper_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - rg2_finger_joint1
        - rg2_finger_joint2
      command_interfaces:
        - position
      state_interfaces:
        - position
        - velocity
      constraints:
        stopped_velocity_tolerance: 0.01
        goal_time: 1.0
      allow_partial_joints_goal: true

